auditlog/__openerp__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     'name': "Audit Log",
-    'version': "8.0.1.2.0",
+    'version': "8.0.1.3.0",
```

Comment:
According to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers I would tend to have 8.0.2.0.0  (new views and new data in models).
What do you think?